packages/docs/docs/rules/banned-dependencies.md

```diff
@@ -2,21 +2,22 @@
 title: bannedDependencies
 ---
 
-Disallow problematic dependencies.
+Prevents the use of specific problematic dependencies within a project.
```

Comment:
Might be useful to remove "problematic" from this sentence and instead have an Overview section similar to what you did for "Alphabetical Dependencies" that explains why the rule is useful. Would also be useful to explain in here whether this is a check of direct dependencies or also transitive dependencies.